frontend/src/components/BigPopup/CommentSection/Comment.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  ListItem,
+  ListItemText,
+  Avatar,
+  ListItemAvatar,
+  Typography,
+  Box,
+  Link
+} from '@mui/material';
+import ThumbUpOutlinedIcon from '@mui/icons-material/ThumbUpOutlined';
+import ThumbDownAltOutlinedIcon from '@mui/icons-material/ThumbDownAltOutlined';
+
+interface Props {
+  text: string;
+  createdAt: Date;
+}
+
+const Comment = ({ text, createdAt }: Props) => {
+  const d = new Date(createdAt);
+  const formatedDate = `${d.getDate()}-${
+    d.getMonth() + 1
+  }-${d.getFullYear()} ${d.getHours()}:${d.getMinutes()}`;
```

Comment:
we need to configure your prettier and linter - because - this part of code hard to read

Reply:
This string is formatted by prettier. Try to format in another way